clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -135,25 +178,46 @@ class ClioWebClient(
     }
   }
 
+  /** Get the version reported by the Clio server. */
   def getClioServerVersion: Source[Json, NotUsed] =
     dispatchRequest(HttpRequest(uri = s"/$versionString"), includeAuth = false)
       .via(parserFlow)
 
+  /** Get the health reported by the Clio server. */
   def getClioServerHealth: Source[Json, NotUsed] =
     dispatchRequest(HttpRequest(uri = s"/$healthString"), includeAuth = false)
       .via(parserFlow)
 
+  /**
+    * POST new metadata to the Clio server for a document key.
```

Comment:
Do you think it's important for the caller to know the underlying http request type for each API? To me that seems like an implementation detail that the caller shouldn't need to know about.